packages/api-sdk/src/medical/models/fhir.ts

```diff
@@ -86,30 +86,25 @@ export const resourceSchema = z.array(z.enum(resourceTypeForConsolidation));
 export const consolidationConversionType = ["html", "pdf", "json"] as const;
 export type ConsolidationConversionType = (typeof consolidationConversionType)[number];
 
-export const consolidatedFilterSchema = z.object({
-  resources: z.string().optional(),
-  dateFrom: z.date().optional(),
-  dateTo: z.date().optional(),
+export const getConsolidatedFiltersSchema = z.object({
+  resources: z.enum(resourceTypeForConsolidation).array().optional(),
+  dateFrom: z.string().optional(),
+  dateTo: z.string().optional(),
+  conversionType: z.enum(consolidationConversionType).default("json").optional(),
 });
-export type ConsolidatedFilter = z.infer<typeof consolidatedFilterSchema>;
+
+export type GetConsolidatedFilters = z.infer<typeof getConsolidatedFiltersSchema>;
 
 export const consolidatedCountSchema = z.object({
   total: z.number(),
   resources: z.record(z.number()),
-  filter: consolidatedFilterSchema,
+  filter: getConsolidatedFiltersSchema.pick({ dateFrom: true, dateTo: true }).extend({
+    resources: z.string(),
```

Comment:
Couldn't we just pick `resources` from `getConsolidatedFiltersSchema` as well?

Reply:
No because the consolidatedFilters is an array of strings and the filter in consolidatedCount is just a string

Reply:
Oh, shoot. Sounds good. 👍